Cartfile

```diff
@@ -0,0 +1,2 @@
+github "goprimer/primer-ios-sdk" ~> 3.1
```

Comment:
Could you please also change this to lock the version? s/`~>`/`==`/

Also, might be worthwhile to add a newline to the end of the file.